bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/ServerConfiguration.java

```diff
@@ -1804,6 +1805,26 @@ public long getAuditorPeriodicBookieCheckInterval() {
         return getLong(AUDITOR_PERIODIC_BOOKIE_CHECK_INTERVAL, 86400);
     }
 
+    /**
+     * Set what percentage of a ledger (fragment)'s entries will be verified.
+     * 0 - only the first and entry of each ledger fragment would be verified
```

Comment:
Is there some typo here?

Reply:
can you clarify a bit more?

Reply:
Is it 'the first entry' ? Dropping the 'and' word

Reply:
okay. that should be "first and last entry"

Reply:
I guess so. Thanks

Reply:
Thank.